### PR TITLE
cutter: Add cutter v2.0.5 (Moved from main)

### PR DESCRIPTION
**Pull request by segevfiner**
Closes N/A (Can open one if needed), related to ScoopInstaller/main#3268

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by segevfiner** [comment]
@rashil2000

**Comment by rashil2000** [comment]
Do we still need the `rz-*.exe` shims in a GUI app? 

(I'm assuming they're the same ones we add in the package in Main).

**Comment by segevfiner** [comment]
Since cutter vendors rizin and we provide a rizin shim it makes sense to me to include them. Rizin/Cutter is quite an in development so there might be a need to use the same version of rizin cutter itself uses.

File: bucket/cutter.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "rz-run.exe"
+    ],
+    "shortcuts": [
+        [
+            "cutter.exe",
+            "Cutter"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/radareorg/cutter"
```
Comment (rashil2000) [inline bucket/cutter.json:31]:
```suggestion
        "github": "https://github.com/rizinorg/cutter"
```

**Pull request merged**